package.json

```diff
@@ -49,6 +49,7 @@
     "vuex": "^3.6.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.11.6",
```

Comment:
I'm not sure it's really useful to describe this dependency in our `package.json` as it is already listed as a dep of `@vue/cli-plugin-babel` and `vue/cli` will make sure the version of `@babel/core` matches the one it was built for.
I'd prefer we let `@vue/cli-plugin-babel` set the correct version than to overwrite ourself.